Assets/Scripts/PlayerController.cs

```diff
@@ -53,6 +55,14 @@ public class PlayerController : MonoBehaviour
     // this is just here for battle of the concepts
     [Header("Temporary")]
     [SerializeField] private GameObject poofSmoke;
+    [FormerlySerializedAs("maxDownSpeed")]
```

Comment:
Looking at the player prefab, it's not serialized as `maxDownSpeed` anymore, so I'm not sure this is still needed.